api/controllers/userController.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response } from "express";
+import userService from "../services/userService";
+import { asyncErrorHandler } from "../middlewares/error";
+
+class user {
+  static googleLogin = (req: Request, res: Response) => {
+    const code: any = req.headers.authorization;
+    const accessToken = userService.googleLogin(code);
```

Comment:
```suggestion
    const code:string = req.headers.authorization;
    const accessToken:string = userService.googleLogin(code);
```

```suggestion
    const code = req.headers.authorization;
    const accessToken = userService.googleLogin(code);
```

type을 둘 다 없애거나 
둘 다 지정하는게 좋을 것 같은데
any로 지정하신 이유가 있나요?